Sources/ElementDiff.swift

```diff
@@ -127,16 +127,16 @@ extension UICollectionView {
 
   /// Update UICollectionView items based on `ElementDiff`
   public func updateSection(section: Int, diff: ElementDiff) {
```

Comment:
Should be `func update(section: Int, diff: ElementDiff)` or `func updateSection(_ section: Int, diff: ElementDiff)`